Makefile

```diff
@@ -280,6 +280,10 @@ tidy-js: check-js-beautify
 tidy: tidy-js
 	tools/tidy
 
+.PHONY: test-containers-compose
+test-containers-compose:
```

Comment:
Hm, that would be the first target that is not included when calling "make test" without any other arguments. I suggest to include it there as well. See how we currently split targets explicitly for different circleCI jobs but still have it included when one calls "make test" locally

Reply:
I was a bit worried to get pushback if I add more to `test` :-D I'll add it then

Reply:
Which would mean you would need my changes first